### ISSUE TITLE
live home page

### ISSUE THREAD

**Issue by jstrachan** [jenkins-x/jx-pipelines-visualizer#92]
rather like #73 it would be nice if the home page refreshed periodically as new pipelines start or complete/fail

**Comment by vbehar** [comment]
yes! it's a bit of work, because right now all the templating is done server-side, and if we want to do that we need to rewrite the templating client-side.
btw you've seen the new `/running` page which shows the running steps in real time?

**Comment by dlidstrom** [comment]
I use Code Injector plugin to inject this:

```javscript
setTimeout(() => window.location.reload(), 10000)
```
